packages/block-editor/src/components/block-list/index.native.js

```diff
@@ -116,7 +117,7 @@ export class BlockList extends Component {
 					accessibilityLabel="block-list"
 					autoScroll={ this.props.autoScroll }
 					innerRef={ this.scrollViewInnerRef }
-					extraScrollHeight={ innerToolbarHeight + 10 }
+					extraScrollHeight={ innerToolbarHeight + globalToolBarHeight + 4 } // + 4 to show the block's bottom border line.
```

Comment:
Maybe give the 4 a proper constant with a name that reflects the comment?

Reply:
A constant sounds good, but the reason is quite subtle.
Does `extraBottomBorderSpace` sound good enough to explain the intention without needing a comment?

Reply:
Maybe `blockBorderSize` or `blockBorderMargin` are clearer?

Reply:
Technically is not the size of the border, but a bit more (that's why the `extra` word). I'll go with the margin option.